src/OrchardCore.Modules/OrchardCore.Alias/Handlers/AliasPartHandler.cs

```diff
@@ -90,9 +81,20 @@ public override async Task CloningAsync(CloneContentContext context, AliasPart p
             clonedPart.Alias = await GenerateUniqueAliasAsync(clonedPart.Alias, clonedPart);
 
             clonedPart.Apply();
+        }
+
+        /// <summary>
```

Comment:
Comments for private method is not required

Reply:
straight out of autoroute. don't see the problem with it myself